src/client/components/localisation/view.tsx

```diff
@@ -3,19 +3,22 @@ import { runInAction } from 'mobx'
 import { observer } from 'mobx-react'
 import * as React from 'react'
 import { WebGLRenderer } from 'three'
+import { lazyInject } from '../../../inversify.config'
```

Comment:
Add a webpack alias for this. Can it also have a better name while you're at it? I'm not fond of it being called a config.